src/Umbraco.Web/PropertyEditors/BlockListConfiguration.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Newtonsoft.Json;
+using Umbraco.Core.PropertyEditors;
+
+namespace Umbraco.Web.PropertyEditors
+{
+
+    /// <summary>
+    /// The configuration object for the Block List editor
+    /// </summary>
+    public class BlockListConfiguration
+    {
+        [ConfigurationField("elementTypes", "Element Types", "views/propertyeditors/blocklist/blocklist.elementtypepicker.html", Description = "Select the Element Types to use as models for the items.")]
+        public ElementType[] ElementTypes { get; set; }
```

Comment:
@Shazwazza I guess this "element type picker" will be a global prevalue editor, which can be re-used in e.g. Nested Content and in third party packages?

Reply:
@bjarnef The "Block Editor" RFC is really stating that we are building the underlying functionality for implementation of block editors. For the first editor, we are creating the Block List editor (name may change) which is what the RFC is talking about. In the future there may be other editors based on the block editor functionality which is what the RFC also says. So yes, in the future part of the code in this PR will be re-used. As for BlockListConfiguration ... maybe, maybe not, this is just the data type structure for the block list editor which as it stands now is incomplete ... this is just stub code until we create the whole thing.